app/views/recipes/_form.html.erb

```diff
@@ -0,0 +1,47 @@
+<%= form_with(model: recipe) do |form| %>
+  <% if recipe.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(recipe.errors.count, "error") %> prohibited this recipe from being saved:</h2>
+
+      <ul>
+        <% recipe.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div>
+    <%= form.label :name, style: "display: block" %>
+    <%= form.text_field :name %>
+  </div>
+
+  <div>
+    <%= form.label :preparation_time, style: "display: block" %>
+    <%= form.number_field :preparation_time %>
+  </div>
+
+  <div>
+    <%= form.label :cooking_time, style: "display: block" %>
+    <%= form.number_field :cooking_time %>
+  </div>
+
+  <div>
+    <%= form.label :description, style: "display: block" %>
+    <%= form.text_area :description %>
+  </div>
+
+  <div>
+    <%= form.label :public, style: "display: block" %>
+    <%= form.check_box :public %>
+  </div>
+
+  <div>
+    <%= form.label :user_id, style: "display: block" %>
+    <%= form.text_field :user_id %>
+  </div>
+
+  <div>
+    <%= form.submit %>
+  </div>
+<% end %>
```

Comment:
- [ ] Hi, please consider checking if the add recipe form works correctly. I'm not able to add a recipe, the form displays the error below yet I've logged In. 
- [ ] Also, I think we can get the user from the `current_user` method provided in `devise` instead of including it in the form.

## Error from Form
<img src="https://user-images.githubusercontent.com/16392046/193904679-aaa8be24-1861-45d8-a834-ee3e49054ec8.png" width=300 />

## Recipes in the database after clicking Create Recipe button
![Screenshot from 2022-10-04 22-02-08](https://user-images.githubusercontent.com/16392046/193904915-af478c61-8ab3-472c-a72f-4ef84c5984a6.png)